.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: java
+jdk:
+  - openjdk7
+  - oraclejdk7
+  - oraclejdk8
+before_install:
+ - chmod +x gradlew
```

Comment:
If on Windows, this might help: http://blog.lesc.se/2011/11/how-to-change-file-premissions-in-git.html